web/src/repogroups/Python2To3.tsx

```diff
@@ -0,0 +1,46 @@
+import { CodeHosts, RepogroupMetadata } from './types'
+
+export const python2To3Metadata: RepogroupMetadata = {
+    title: 'Refactor Python 2 to 3',
+    name: 'python-2-to-3',
+    url: '/refactor-python2-to-3',
+    repositories: [
+        { name: 'sourcegraph/sourcegraph', codehost: CodeHosts.GITHUB },
+        { name: 'python/python', codehost: CodeHosts.GITHUB },
+        { name: 'test/test', codehost: CodeHosts.GITHUB },
+        { name: 'sourcegraph/src-cli', codehost: CodeHosts.GITHUB },
+        { name: 'sourcegraph/sourcegraph', codehost: CodeHosts.GITLAB },
+        { name: 'test/test', codehost: CodeHosts.GITLAB },
+        { name: 'test/test2', codehost: CodeHosts.GITLAB },
+        { name: 'sourcegraph/src-cli', codehost: CodeHosts.GITHUB },
+        { name: 'sourcegraph/src-cli', codehost: CodeHosts.BITBUCKET },
+    ],
+    description: `This repository group contains python 2 and 3 repositories and corresponding search examples.
+                        The search examples will help you find code that requires refactoring and review example Python
+                        3 syntax.`,
+    examples: [
+        {
+            title: 'Python 2 imports',
+            exampleQuery: '"from :[package.] import :[function.]”',
```

Comment:
There's a fancy quote in this string
```suggestion
            exampleQuery: '"from :[package.] import :[function.]"',
```